src/main/java/edu/hm/hafner/coverage/Metric.java

```diff
@@ -32,6 +32,11 @@ public enum Metric {
     BRANCH(new ValuesAggregator()),
     INSTRUCTION(new ValuesAggregator()),
 
+    /* VectorCAST Coverage */
+    MCDC_PAIR(new ValuesAggregator()),
+    FUNCTION(new ValuesAggregator()),
```

Comment:
I think it would make sense to remove this metric as and reuse `METHOD`.

Reply:
I think that I also found the problem why the `METHOD` metric does not work: It seems that your coverage.xml files do not map the lines to methods, the line nodes are just reported per file, and the methods are empty. I'll check if I can change the evaluation of the METHOD value on my side.

Reply:
I can either change the metrics currently named "function-coverage" to something else "method-coverage" or I can add the lines to the method instead or along with the file.  Your choice.

Would you like me to rename the VectorCASTParser to ExtendedCoberturaParser?

Reply:
> I can either change the metrics currently named "function-coverage" to something else "method-coverage" or I can add the lines to the method instead or along with the file. Your choice.

I am not sure if you can map the lines to the methods. They are not in the report file so I don't know how you can achieve that? 

Extract from your test file:
```
<methods>
            <method name="Get_Record" signature="(int)data_type" line-rate="1.0" branch-rate="1.0" complexity="2" functioncall-coverage="100.00% (2/2)" mcdcpair-coverage="100.00% (1/1)" function-coverage="100% (1/1)"/>
            <method name="Update_Record" signature="(int,data_type)" line-rate="1.0" branch-rate="0.6" complexity="2" functioncall-coverage="100.00% (1/1)" mcdcpair-coverage="0.00% (0/1)" function-coverage="100% (1/1)"/>
            <method name="Remove_Record" signature="(int)" line-rate="1.0" branch-rate="1.0" complexity="1" functioncall-coverage="100.00% (1/1)" function-coverage="100% (1/1)"/>
          </methods>
          <lines>
            <line number="45" hits="1" branch="true" condition-coverage="100.0% (1/1)">
              <conditions>
                <condition number="0" type="jump" coverage="100.0%"/>
              </conditions>
            </line>
            <line number="48" hits="1" branch="false"/>
            <line number="51" hits="1" branch="false"/>
            <line number="54" hits="1" branch="false" functioncall-coverage="100.0% (1/1)"/>
```
There is no way to see which lines are in which method, or am I missing something?

In Java Cobertura reports you have:

```
<method name="method">
<line number="48" hits="1" branch="false"/>
            <line number="51" hits="1" branch="false"/>
            <line number="54" hits="1" branch="false" functioncall-coverage="100.0% (1/1)"/>
</ method >
```

Reply:
> Would you like me to rename the VectorCASTParser to ExtendedCoberturaParser?

No, the parser is actually a VectorCAST specific parser, this is fine. I think just the metrics and the model should not contain a tool name. Maybe we can use the correct Java capitalization which uses CamelCase even for short forms: `VectorCast`.

Reply:
done

Reply:
Do you need some help with the model so we can eliminate the duplicate enum `FUNCTION`? It would be very helpful if we can omit this value and use the `METHOD` coverage.